serverlessworkflow/modules/ROOT/pages/security/orchestrating-third-party-services-with-oauth2.adoc

```diff
@@ -456,8 +456,11 @@ Also, you can use the alternatives defined in the link:{quarkus_guide_config_ref
 Once you clone the `serverless-workflow-oauth2-orchestration-quarkus` example application from GitHub repository, you can run the example application.
 
 .Prerequisites
-* You have an account in GitHub.
-* Apache Maven {maven_min_version} or later is installed.
+* link:{java_install_url}[Java] {java_min_version} is installed.
+* link:{maven_install_url}[Maven] {maven_min_version} or later is installed.
+* link:{docker_install_url}[Docker] {docker_min_version} or later is installed.
+* (Optional) link:{docker_compose_install_url}[Docker compose] {docker_compose_min_version} or later is installed.
```

Comment:
nitpicking but is there a way to use podman instead ?

Reply:
Not sure it's a big win here. In general around the other examples, and guides, that use for example minikube the container platform used is Docker.

Reply:
fair enough